web/domain/dao/data_processing_dao.py

```
@@ -0,0 +1,279 @@
+
+
+            existing_file = result_file.scalars().first()
+
+
+
+
+
+            if not existing_file:
+
+
+                print(f"[DAO]   -> No existing DisclosureFile. Creating new one.")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)